### PR TITLE
Add usage guide

### PR DESCRIPTION
**Pull request by GMartigny**
#### Information 
 - Kind: Guide
 - Emoji that best describe this PR: :memo: 

#### Description 
Add a guide to help people use the library.

### CODE REVIEW

File: guide/index.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        Or you can load it with NPM:
+    </p>
+    <pre><code>
+$ npm install pencil.js
+        </code></pre>
+    <pre><code class="javascript">
+import Pencil from "pencil.js";
+    </code></pre>
+    <p>
+        In either case, you can now use it:
```
Comment (Heraclite) [inline guide/index.html]: both ?

File: guide/index.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    <h2>Components</h2>
+    <p>
+        {Component} refers to any drawable part of a scene.
+        As an abstract class, it should not be instantiated by itself, but by any sub-class (ex: Rectangle, Star, Image &hellip;).
+    </p>
+    <pre><code class="javascript">
+const myRectangle = new Pencil.Rectangle();
+    </code></pre>
+    <p>
+        All sub-class constructor parameters are as follows: <code class="inline">(position, [...a number of specific properties], options)</code>.<br/>
```
Comment (Heraclite) [inline guide/index.html]: All components can be instanciated as follow ?

File: guide/index.html [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        To continue on our previous example:
+    </p>
+    <pre><code class="javascript">
```
Comment (Heraclite) [inline guide/index.html]: That would be nice to see the result. Even if that's already in the presentation page.

File: guide/index.html [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+        {Position} is an utility class used all across Pencil.js library.<br/>
+        It serves as a wrapper for a x and y coordinate in 2D space. You can create one using the class constructor.
+    </p>
+    <pre><code class="javascript">
+const x = 100;
+const y = 200;
+const myPosition = new Position(x, y);
+    </code></pre>
+    <p>
+        Alternatively, anywhere a position is needed, you can use an useful array shorthand.
```
Comment (Heraclite) [inline guide/index.html]: you can use this nice || simple || intuitive shorthand ?

**Comment by GMartigny** [comment]
> For long text, maybe a more readable font should be use (Open sans, Roboto).

I tried ['Open sans', 'Roboto' and 'Source Sans Pro'](https://fonts.google.com/selection?selection.family=Montserrat|Open+Sans|Roboto|Source+Sans+Pro). The last one feel a bit more condensed and professional.

I also tried to limit paragraphs size and justify them. It looks odd, but I don't know if it's a good or bad thing.

> Yellow as background is a mistake :) !

Yellow backgrounds are gone ;)

I kept a little separation between screens with a tiny drop-shadow. What do you think ?

> we have to found a way to make the sidebar more readable

What do you mean by "more readable" ?
I tried adding a drop-shadow (to stay in the same theme as screens).

File: guide/index.html (hunk 3 above)
Comment (GMartigny) [reply guide/index.html]: I don't know how to do it. It's a good idea, but too much preview would impair reading flow.

**Pull request closed without merging**